library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +71,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    Set<DayViewDecorator> dayViewDecorators;
```

Comment:
I chose a set to avoid duplication of decorators when adding/removing dynamically. What do you think about changing the current implementation (HashSet) by a [LinkedHashSet](http://developer.android.com/reference/java/util/LinkedHashSet.html)? This implementation 
ensures predictable iteration order (insertion-order), maintaining the Set advantages.